frontend/src/components/profile/profile.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { SideMenu, UserInfo, Documents } from './components';
+import { EditUserPopup } from 'components/common';
+import { IEditUserPayload, IUserTypeDoctor } from 'common/interfaces';
+import { UserType } from 'common/enums';
+import { AuthActionCreator } from 'store/slices';
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { RootState, UserGeneric } from 'common/types';
+import { useParams } from 'react-router';
+import { UsersActionCreator } from 'store/slices';
+
+import styles from './styles.module.scss';
+
+type RouteParam = {
+  id:string
+};
+
+const Profile: React.FC = () => {
+  const [isEditeMode, setIsEditMode] = React.useState<boolean>(false);
+  const [user, setUser] = React.useState<UserGeneric>();
```

Comment:
```suggestion
  const [user, setUser] = React.useState<user>();
```
We need to create a new slice for it and don't share the logic with `user` slice. So remove all `isCurrent user` and other connected things with this